math/src/number_128.rs

```diff
@@ -72,15 +72,33 @@ impl Number128 {
         Self::from_decimal(basis_points, crate::BPS_EXPONENT)
     }
 
-    /// Get the underlying 128-bit representation
-    pub fn into_bits(self) -> i128 {
-        self.0
+    /// Get the underlying 128-bit representation in bytes.
+    /// Uses the target endianness of the caller
+    pub fn into_bits(self) -> [u8; 16] {
+        #[cfg(target_endian = "little")]
+        return self.0.to_le_bytes();
+        #[cfg(target_endian = "big")]
+        self.0.to_be_bytes()
```

Comment:
couldn't you just use `to_ne_bytes` ?

Reply:
i think most of the solana runtime uses little endian, if that matters.